### ISSUE TITLE
Change the default font in the Task area of Text From Subject tasks to an unambiguous font

### ISSUE THREAD

**Issue by seanmiller26** [zooniverse/front-end-monorepo#4195]
## Is your feature request related to a problem? Please describe.
Volunteers need to be able to distinguish differences between commonly confused letters such as 1, l, and I. This is crucial in transcribing OCR work.

## Describe the solution you'd like
Replace the default font 'Karla' with 'Anonymous Pro'. 

Available from Google here: https://fonts.google.com/specimen/Anonymous+Pro?preview.text=O%200%20I%20l%201&preview.text_type=custom

<img width="954" alt="Screen Shot 2023-02-13 at 11 25 26 AM" src="https://user-images.githubusercontent.com/89043489/218528609-9514e4e4-3c62-436a-bb0f-12b36372466e.png">

As discussed, it should be able to be defined here: https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/plugins/tasks/experimental/textFromSubject/components/TextFromSubjectTask.js#L49-L57
